react/features/conference/components/web/Subject.js

```diff
@@ -96,7 +96,7 @@ function _mapStateToProps(state) {
         _showParticipantCount: participantCount > 2 && !hideParticipantsStats,
         _showSubject: !hideConferenceSubject,
         _subject: getConferenceName(state),
-        _visible: isToolboxVisible(state) && participantCount > 1
```

Comment:
This change seems unrelated.

Reply:
Not necessarily, this PR enables the subject to be visible at the same time as the `InviteMore` component so we need this in order for that to happen.

Reply:
Ack